fuse/eval/metrics/sequence_gen/metrics_seq_gen_common.py

```diff
@@ -106,6 +117,10 @@ def _perplexity_compute(
     log_probs: List[np.ndarray],
     token_num: List[np.ndarray],
 ) -> float:
+    # avoid from overflow on large epochs
+    log_probs = [e.astype(np.float64) for e in log_probs]
```

Comment:
numerical stability?

Reply:
It might be large numbers with many GPUS so I took the safe side here.